app/components/avo/index/grid_item_component.rb

```diff
@@ -2,7 +2,6 @@
 
 class Avo::Index::GridItemComponent < ViewComponent::Base
   include Avo::ResourcesHelper
-
```

Comment:
Can we leave this empty space please? It sets a boundary between the different types of declarations.

```suggestion

```